src/cf-forms/src/molecules/field.less

```diff
@@ -22,6 +22,7 @@
                 width: unit( 18px / @base-font-size-px, em );
                 margin-right: 10px;
 
+                background-color: @input-bg;
                 content: '';
                 vertical-align: top;
```

Comment:
Good catch, I didn't even notice this when I had done the original checkboxes. Your suggestion fixes it for both normal and large target chekboxes and looks good across browsers.